en/views/helpers/form.rst

```diff
@@ -1518,84 +1522,11 @@ way PHP handles data passed via file fields
-  added inside ``select`` at the top of the list. If a string, that string is
-  displayed as the empty element. Defaults to ``true``.
-
-* ``'default'`` | ``value`` - Use either of the two to set the default value to
-  be shown by the field. A value in ``$this->request->getData()`` matching the field
-  name will override this value. If no default is provided ``time()`` will
-  be used.
-
-* ``'year', 'month', 'day', 'hour', 'minute', 'second', 'meridian'`` - These
-  options allow you to control which control elements are generated or not.
-  By setting any of these options to ``false`` you can disable the generation
-  of that specific that select picker (if by default it would be rendered in
-  the used method). In addition each option allows you to pass HTML attributes
-  to that specific ``select`` element.
-
-.. _date-options:
-
-Options for Date-Related Controls
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-These options are concerning the date-related methods - i.e. ``year()``,
-``month()``, ``day()``, ``dateTime()`` and ``date()``:
-
-* ``'monthNames'`` - If ``false``, 2 digit numbers will be used instead of text
-  for displaying months in the select picker. If set to an array (e.g.
-  ``['01' => 'Jan', '02' => 'Feb', ...]``), the given array will be used.
-
-* ``'minYear'`` - The lowest value to use in the year select picker.
-
-* ``'maxYear'`` - The maximum value to use in the year select picker.
-
-* ``'orderYear'`` - The order of year values in the year select picker.
-  Possible values are ``'asc'`` and ``'desc'``. Defaults to ``'desc'``.
-
-.. _time-options:
-
-Options for Time-Related Controls
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-These options are concerning the time-related methods - ``hour()``,
-``minute()``, ``second()``, ``dateTime()`` and ``time()``:
-
-* ``'interval'`` - The interval in minutes between the values which are
-  displayed in the ``option`` elements of the minutes select picker.
-  Defaults to 1.
-
-* ``'round'`` - Set to ``up`` or ``down`` if you want to force rounding minutes
-  in either direction when the value doesn't fit neatly into an interval.
-  Defaults to ``null``.
-
-* ``timeFormat`` - Applies to ``dateTime()`` and ``time()``. The time format to
-  use in the select picker; either ``12`` or ``24``. When this option is set to
-  anything else than ``24`` the format will be automatically set to ``12`` and
-  the ``meridian`` select picker will be displayed automatically to the right of
-  the seconds select picker. Defaults to 24.
-
-* ``format`` - Applies to ``hour()``. The time format to use; either ``12`` or
-  ``24``. In case it's set to ``12`` the ``meridian`` select picker won't be
-  automatically displayed. It's up to you to either add it or provide means
-  to infer from the form context the right period of the day. Defaults to 24.
-
-* ``second`` - Applies to ``dateTime()`` and ``time()``. Set to ``true`` to
-  enable the seconds drop down. Defaults to ``false``.
-
```

Comment:
Do you have 2 headings in a with this block removed?

Reply:
Not sure what you mean.

Reply:
From the diff it looks like there will be two adjacent headings. I can check that post merge though.